src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, partnerName, isMarried, sex } = person;
+
+  // prettier-ignore
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+      )}
+      {isMarried ? (
+        <p className="Person__partner">
+          {`${partnerName} is my `}
+          {sex === 'm' ? 'wife' : 'husband'}
+        </p>
+      ) : (<p className="Person__partner">I am not married</p>)}
```

Comment:
Looks too complicated and also try not to use nested ternary operator, instead you can move this part - sex === 'm' ? 'wife' : 'husband' to separate variable partner.

Also you can use only one `<p>` tag and use ternary only for content